pulpcore/download/base.py

```diff
@@ -97,6 +102,17 @@ def __init__(
         self._digests = {n: pulp_hashlib.new(n) for n in Artifact.DIGEST_FIELDS}
         self._size = 0
 
+        if expected_digests:
+            bad_digests = [
+                key for key in self.expected_digests.keys() if key in Artifact.FORBIDDEN_DIGESTS
+            ]
+            if bad_digests:
+                raise UnsupportedDigestValidationError(
+                    _("Cannot download artifact at '{}' due to bad digests: {}").format(
```

Comment:
```suggestion
                    _("Cannot download Artifact at '{}' due to bad digests: {}").format(
```

maybe this tho ^? I'm not sure.

Reply:
@daviddavis wdyt about this ^? My motivation is to have the user understand that it's not bad like invalid, but bad as in blocklisted and how they can change it.